packages/@okta/vuepress-site/docs/concepts/role-assignment/index.md

```diff
@@ -0,0 +1,115 @@
+
+Standard role assignment is done in two steps:
+1. Assign a Role to a User or Group. At this point the admin has the supported privileges of the role over all resources across organization.
+2. (Optional) If the role supports targets, use one of the [target operations](/docs/reference/api/roles/#role-target-operations) to specify which specific resource the admin can manage.
+
+Note that the entities involved in standard role assignment are:
+* a Role; identified either by type or id returned from [listing API](/docs/reference/api/roles/#list-roles)
+* a Principal; either a Group or a User
+* (Optional) a resource; when using [target operations](/docs/reference/api/roles/#role-target-operations) this could be either an App or a Group
+
+## Custom Role Assignment
+Custom roles can be built by piecing [permissions](/docs/reference/api/roles/#permission-types) together. Once a custom role is built, you can use its `id` or `label` to assign to admins. The process is:
+1. Build a custom role
+2. Build a set of resources
+3. Bind the admin with the role from step 1 targeting the resource set from step 2
+
+An assignment of a Role to an admin is called a [binding](/docs/reference/api/roles/#binding-object). Conceptually, a Binding is identified by a unique id, and represents a single unique combination of Principal, Resource-Set, and Custom Role. A given Resource-Set can have multiple bindings, allowing for different combinations of principals and roles as necessary to grant permission to the encompassing resource.
+
+Therefore, when dealing with custom roles, three entities are always in play:
+* a Role; identified by its `label` or `id`
+* a Principal; either a Group or a User - known as a Member of the binding
+* a Resource-Set; identified by its `id`
+
+### Resource Sets
+A Resource Set is simply a collection of resources. The following resources are currently supported:
+* All users
+* All groups
+* A specific group
+* All users within a specific group
+
+### Identifiers
+#### Resource Identifiers
+In order to specify a resource targeted by a resource set, you will simply use the REST URL of the corresponding Okta API:
+* [All users](/docs/reference/api/users/#list-users)
+  ``` http
+  https://${yourOktaDomain}/api/v1/users
+  ```
+* [All groups](/docs/reference/api/groups/#list-groups)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups
+  ```
+* [A specific group](/docs/reference/api/groups/#get-group)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}
+  ```
+* [All users within a specific group](/docs/reference/api/groups/#list-group-members)
+  ``` http
+  https://${yourOktaDomain}/api/v1/groups/${targetGroupId}/users
+  ```
+
+> **Note:** All users and groups could be represented using either of [All users](/docs/reference/api/users/#list-users) or [All groups](/docs/reference/api/groups/#list-groups) links. There is no functional difference between the two.
```

Comment:
"All users and groups can be represented using either [All users] or [All groups] links." < replace "could" with "can" and remove "of"